SDL/PixelTriplet.cu

```diff
@@ -642,12 +642,13 @@ __device__ float SDL::computePT3RPhiChiSquared(struct modules& modulesInGPU, uin
         }
     }
     // this for loop is kept to keep the physics results the same but I think this is a bug in the original code. This was kept at 5 and not nPoints
-    for(size_t i = 3; i < 5; i++)
-    {
-        delta1[i] /= 0.009f;
-        delta2[i] /= 0.009f;
-    }
-    chiSquared = computeChiSquaredpT5(3, xs, ys, delta1, delta2, slopes, isFlat, g, f, radius); //unclear why but the pT3 version doesn't work despite being copy and pasted
+    //for(size_t i = 3; i < 5; i++)
```

Comment:
These are the lines that caused the bug

Reply:
Using any computeChiSquared function with these lines removed gives same results as using the inlined functions.